src/scripts.js

```diff
@@ -2,107 +2,209 @@ import './css/styles.css';
-        </div>
-        <img class="trip-img" src=${tripDestination.image} alt=${tripDestination.alt} width="300" height="205">
-      </div>
-      <div class="trip-info">
-        <p class="destination">${tripDestination.destination}</p>
-        <p>${trip.date} - ${endDate}</p>
-        <p>Estimated Cost: $${tripTotalCost}</p>
-        <p>Guests: ${trip.travelers}</p>
-      </div>
+    <div class="photo">
+    <div class="status-label ${tripTimeEra}">
+    <p>${tripTimeEra}</p>
+    </div>
+    <img class="trip-img" src=${tripDestination.image} alt=${tripDestination.alt} width="300" height="205">
+    </div>
+    <div class="trip-info">
+    <p class="destination">${tripDestination.destination}</p>
+    <p>${trip.date} - ${endDate}</p>
+    <p>Estimated Cost: $${tripTotalCost}</p>
+    <p>Guests: ${trip.travelers}</p>
+    </div>
     </section>`
   })
-
+  
   tripsDisplay.innerHTML = tripsData
 }
 
 function displayTotalCostThisYear(totalCost) {
   costThisYear.innerText = `$${totalCost}`
 }
 
+//FORM 
+
 function displayDestinationOptions(destinations) {
   let options = ''
   destinations.forEach(destination => {
     options += 
-    `<option value="${destination.destination}">${destination.destination}</option>`
+    `<option value="${destination.id}">${destination.destination}</option>`
   })
-  destinationsPicker.innerHTML = options
+  destinationsPicker.innerHTML += options
 }
 
 function preventPickingPastDates() {
-  const today = Date()
-  const todayWithFormat = dayjs(today).format('YYYY-MM-DD')
-  datePicker.innerHTML += `<input class="date-picker" type="date" min="${todayWithFormat}"></input>`
+  today = dayjs(Date()).format('YYYY-MM-DD')
+  datePicker.min = dayjs(today).add(1, 'day').format('YYYY-MM-DD')
 }
 
 function buildDropDown() {
   let numbers = ''
   for (let i = 1; i < 51; i++) {
     numbers += `<option value="${i}">${i}</option>`
   }
-  durationPicker.innerHTML = numbers 
-  guestPicker.innerHTML = numbers
+  durationPicker.innerHTML += numbers 
+  guestPicker.innerHTML += numbers
+}
+
+//WORK WITH FORM
+
+function checkInput() {
+  console.log(today)
+  if(destinationsPicker.value && datePicker.value > today && durationPicker.value && guestPicker.value) {
+    button.disabled = false
+  }
+}
+
+function checkDate() {
+  if(datePicker.value < today){
+    messageBanner.innerText = 'Opps, choose a date in the future!'
+  }
+}
+
+let clickCounter = 0
+function checkNumberOfClick(){
+  clickCounter ++
+  if(clickCounter === 1){
+    estimateCost()
+  } else if (clickCounter === 2) {
+    postTrip(newPendingTrip)
+  }
+}
+
+function estimateCost() {
+  event.preventDefault()
+  newPendingTrip = { 
+    id: allTrips.tripsRepo.length + 1,
+    userID: displayedUsersID,
+    destinationID: Number(destinationsPicker.value),
+    travelers: guestPicker.value,
+    date: dayjs(datePicker.value).format('YYYY/MM/DD'),
+    duration: durationPicker.value, 
+    status: 'pending',
+    suggestedActivities: []
+  } 
```

Comment:
Good job hardcoding as much of this data as possible. This helps cuts down on potential errors by the user